docs/source/blogs/tech_blog/blog10_ADP_Balance_Strategy.md

```
@@ -0,0 +1,207 @@
+</div>
+<p align="center"><sub><em>Figure 2: Token distribution of input and output token numbers </em></sub></p>
+
+The input and output token ranges are wide, and the output tokens have a clear long-tail distribution. This makes it difficult to co-schedule as many contexts as possible within the same step while avoiding additional bubbles, posing a significant challenge to our scheduling strategy.
+
+- Server
+
+The experiments were conducted using the DeepSeekV3 model on a Blackwell GB200 machine, 8 GPU, with a DP size of 8 for the Attention module and an EP size of 8 for the MoE module.
+
+
+### Experimental Results
+The experiments are divided into three groups: a baseline, ADP Balance with ``timeout_iters` (“Context Wait”), and ADP Balance with both `timeout_iters` and `batching_wait_iters`. Enabling Context Wait provides a significant TPS improvement; adding `batching_wait_iters` further enhances balance.
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)